### ISSUE TITLE
Embed Feedback & Request forms on BizFriendly pages

### ISSUE THREAD

**Issue by arielkennan** [codeforamerica/bizfriendly-web#170]
Instead of going out to a link, embed on a page with a BizFriend.ly header, similar to the About page. Should end up with /feedback.html and /request.html. All nav bar links would need to be updated too. 

```
Content request embed: 
<iframe src="https://docs.google.com/forms/d/1walyyOEbAibc3lVtJgkyaMib7oHvfvEu7zfSCROzh70/viewform?embedded=true" width="760" height="500" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
```

```
Feedback embed: 
<iframe src="https://docs.google.com/forms/d/1-0s2k_gpkNDXWRpYYbzltFNaRT0FEOv2--gIwxGGZWU/viewform?embedded=true" width="760" height="500" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
```

**Comment by arielkennan** [comment]
Also add submitted content requests 

```
<iframe width='500' height='300' frameborder='0' src='https://docs.google.com/spreadsheet/pub?key=0Ai1uBoejCmYrdEZFdFZMamdIdUZobE1hUmtJUDg1eFE&single=true&gid=0&output=html&widget=true'></iframe>
```

**Comment by nickjellings** [comment]
Things should be good to go with pull request #181 

I might suggest stylizing the Google Doc forms a bit by removing the background.

**Comment by ondrae** [comment]
Great. I made the form background white, as you suggested. Looks much better.

**Comment by nickjellings** [comment]
Wanted to mention the background color on the "Content Request" form as
well. Looks like it's still got that default pattern going on. :)

http://bizfriend.ly/request.html

Thanks!

On Tue, Sep 16, 2014 at 7:34 PM, Andrew Hyder notifications@github.com
wrote:

> Closed #170 https://github.com/codeforamerica/bizfriendly-web/issues/170
> .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/codeforamerica/bizfriendly-web/issues/170#event-166381707
> .

## 

Nick Jellings // 563-379-4746 (cell) // nickjellings@gmail.com
